### ISSUE TITLE
[Quest][ZulDrak] Throwing Down

### ISSUE THREAD

**Issue by TheRealestMcCoy** [dalaranwow/dalaran-wow#4529]
**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
First issue is some of the spider dens arent counting. 
The spiderlings are already out side and targetable.
**Expected behaviour**: Tell us what should happen instead
Throwing  grenade in the den should cause it to collapse/explode. then the little spiders should run out and run around then die. The den should then collapse and disappear.
**Steps to reproduce the problem**:

1. Pick up quest 
2. Try using bomb on a spwaned den
3. 

**Include proofs for this behaviour**
![wowscrnshot_030517_153444](https://cloud.githubusercontent.com/assets/12252441/23591110/4b85e9c4-01b9-11e7-891e-e2feeae6819f.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
You can see in the video link the dens do despawn after a point the spider ling event ends.
[Video Link](https://www.youtube.com/watch?v=z_4eKtgDDvg)

Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
[Quest](http://www.wowhead.com/quest=12598/throwing-down)

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
DB/Quest: Throwing Down - Add questevent with Burning Skimmer

* once the Nethurbian Crater KC Bunny was hit by the bomb, Skimmers will spawn and moverandom with aura 'Immolation' for 10 seconds
* add 'Camera Shake - Med' spell once the bomb landed
* add despawn for Crater for ~3 minutes
* decrease spellfocusrange for spellcast: 'Toss Grenade' to match retail value
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4529
* commit: 430308fe0c69923fc42f2b4b2ff36b8a97abb161

![GitHub Logo](http://i.imgur.com/Zl9WfqB.jpg)
![GitHub Logo](http://i.imgur.com/RgLmY0r.jpg)
![GitHub Logo](http://i.imgur.com/ZkPam9Q.jpg)

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
